run-uts

```diff
@@ -9,22 +9,39 @@ else
     SKIP="vendor,$SKIP"
 fi
 
+echo "Removing old coverprofiles..."
+find . -name "*.coverprofile" -type f -delete
+
+echo "Calculating packages to cover..."
+go_dirs=$(find -type f -name '*.go' | \
+	      grep -vE '/vendor/|\./proto/|.glide|/k8sfv/' | \
```

Comment:
I think this (welcome) fix-up belongs in its own PR.

Aren't these exclusions felix-specific?